rl_algorithms/common/networks/resnet.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+"""ResNet modules for RL algorithms.
+
+- Authors: Kh Kim & MS Kim
+- Contacts: kh.kim@medipixel.io
+            minseop.kim@medipixel.io
+"""
+
+from typing import Union
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from rl_algorithms.common.networks.mlp import MLP
+from rl_algorithms.utils.config import ConfigDict
+
+device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+
+
+class BasicBlock(nn.Module):
```

Comment:
should add a simple description like other classes.

Reply:
reference: https://github.com/medipixel/rl_algorithms/blob/27e5ac826a7c34d43ffbe6435ce51bf39aef3a88/rl_algorithms/dqn/networks.py#L76